### PR TITLE
Add Art Collage To Homepage

### LINKED ISSUE #19: Home page layout
Going off of our wireframe, stub out the content and chart / image placeholders for the home page: https://app.moqups.com/pcSQvUMmsyAa1SN58KaKg1EKuYRs8iRX/view/page/a68639957

Screen Shot 2022-03-08 at 4 59 41 PM

### PR DESCRIPTION
**Pull request by vkoves**
## Overview

Adds a collage of our new artwork to our homepage, to give it some fun and flair.

| Mobile | Desktop |
| --- | --- |
| Screenshot from 2022-08-30 20-47-24 | Screenshot from 2022-08-30 20-48-36 |

**Note:** This collage might not be perfect (we can add wind turbines later for instance) but this was using only the assets I already had imported in my previous work and I think it gets us 80% there.

Closes #19 

### Demo

See description.

### Notes

None

## Testing Instructions

* Load up the Netlify and confirm the art looks good on any screen size (not just mobile and desktop but slide the window size around)

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -1,18 +1,72 @@
...
+    justify-content: space-between;
+    margin-top: 3rem;
+    margin-bottom: 6rem;
+}
+
+.art-row * {
+    filter:  drop-shadow(-0.25rem 0.25rem 0rem var(--light-shadow-color));
+}
+
+.art-row .appliance-sheet, .art-row .car-sheet {
```
Comment (vkoves) [inline src/styles/index.css:30]: Man I really need to get Sass setup so I can do nesting and not have to right this cludgy CSS 😆

**Comment by jpvelez** [comment]
I would remove the apartment building, replace the electric truck with an electric cars, move the induction stove next to the boiler, and make the home/vehicle/appliances bigger and a bit more spaced out.

**Comment by derekeder** [comment]
@vkoves this is looking really good! I agree with all of @jpvelez's comments about the composition of the home page image.

**Comment by vkoves** [comment]
@jpvelez and @derekeder - updated, let me know how this looks:

Screenshot from 2022-08-30 20-12-32

**Review by derekeder — APPROVED** [review]
YES

**Review by derekeder — APPROVED** [review]

**Pull request merged**